ext4: fix error ext number comments

From: Shen Feng <shen@cn.fujitsu.com>

Change second/third to fourth.

Signed-off-by: Shen Feng <shen@cn.fujitsu.com>
Signed-off-by: Mingming Cao <cmm@us.ibm.com>
---
 fs/ext4/ext4.h    |    2 +-
 fs/ext4/ext4_i.h  |    2 +-
 fs/ext4/ext4_sb.h |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

Index: linux-2.6.26-rc2/fs/ext4/ext4.h
===================================================================
--- linux-2.6.26-rc2.orig/fs/ext4/ext4.h	2008-05-16 11:55:10.000000000 -0700
+++ linux-2.6.26-rc2/fs/ext4/ext4.h	2008-05-16 11:55:15.000000000 -0700
@@ -22,7 +22,7 @@
 #include "ext4_i.h"
 
 /*
- * The second extended filesystem constants/structures
+ * The fourth extended filesystem constants/structures
  */
 
 /*
Index: linux-2.6.26-rc2/fs/ext4/ext4_i.h
===================================================================
--- linux-2.6.26-rc2.orig/fs/ext4/ext4_i.h	2008-05-11 17:09:41.000000000 -0700
+++ linux-2.6.26-rc2/fs/ext4/ext4_i.h	2008-05-16 11:55:15.000000000 -0700
@@ -79,7 +79,7 @@ struct ext4_ext_cache {
 };
 
 /*
- * third extended file system inode data in memory
+ * fourth extended file system inode data in memory
  */
 struct ext4_inode_info {
 	__le32	i_data[15];	/* unconverted */
Index: linux-2.6.26-rc2/fs/ext4/ext4_sb.h
===================================================================
--- linux-2.6.26-rc2.orig/fs/ext4/ext4_sb.h	2008-05-16 11:16:55.000000000 -0700
+++ linux-2.6.26-rc2/fs/ext4/ext4_sb.h	2008-05-16 11:55:15.000000000 -0700
@@ -25,7 +25,7 @@
 #include <linux/rbtree.h>
 
 /*
- * third extended-fs super-block data in memory
+ * fourth extended-fs super-block data in memory
  */
 struct ext4_sb_info {
 	unsigned long s_desc_size;	/* Size of a group descriptor in bytes */
